README.md

```diff
@@ -114,6 +114,8 @@ For separator you can add thousandSeparator
 | 000.000.000-99 | 048.457.987-98 |
 | AAAA           | 0F6g           |
 | SSSS           | asDF           |
+| UUUU           | ASDF           |
+| SSSS           | asdf           |
```

Comment:
@biel-correa I think you meant to add  `LLLL`  instead of `SSSS` (which already exists)

Reply:
Yes, thanks for warning me